dashboard/app/controllers/api/v1/section_instructors_controller.rb

```diff
@@ -79,6 +79,8 @@ def check
       return render json: {error: 'inviting self'}, status: :bad_request
     end
 
+    puts current_user.inspect
```

Comment:
Oops?

Reply:
Left over from testing, not sure how I missed it XD

Reply:
We all do it, no worries.